nil-pointer error when adding service-registry artifact role

## Description

**Version**: dev

When adding a role to a service-registry, if I do not provide the role value then I get a nil-pointer error.

### Steps to reproduce

```
$ rhoas service-registry role add --username ephelan_kafka_devexp
Updating role for principal
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0xad221b]

goroutine 1 [running]:
github.com/redhat-developer/app-services-cli/pkg/cmd/registry/artifact/role/add.runAdd(0xc0000e0a00, 0xc0000bdd90, 0x0)
        /home/ephelan/code/github.com/redhat-developer/app-services-cli/pkg/cmd/registry/artifact/role/add/add.go:126 +0x2bb
github.com/redhat-developer/app-services-cli/pkg/cmd/registry/artifact/role/add.NewAddCommand.func1(0xc0000dd180, 0xc000557840, 0x0, 0x2, 0x0, 0x0)
        /home/ephelan/code/github.com/redhat-developer/app-services-cli/pkg/cmd/registry/artifact/role/add/add.go:93 +0xe5
github.com/spf13/cobra.(*Command).execute(0xc0000dd180, 0xc000557820, 0x2, 0x2, 0xc0000dd180, 0xc000557820)
        /home/ephelan/.gvm/pkgsets/go1.16.4/global/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:856 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc000369680, 0x2013038, 0x3, 0xc000369680)
        /home/ephelan/.gvm/pkgsets/go1.16.4/global/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:974 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /home/ephelan/.gvm/pkgsets/go1.16.4/global/pkg/mod/github.com/spf13/cobra@v1.2.1/command.go:902
main.main()
        /home/ephelan/code/github.com/redhat-developer/app-services-cli/cmd/rhoas/main.go:52 +0x2aa
```

--- a/pkg/cmd/registry/artifact/role/add/add.go
+++ b/pkg/cmd/registry/artifact/role/add/add.go
@@ -99,6 +99,7 @@ func NewAddCommand(f *factory.Factory) *cobra.Command {
 	cmd.Flags().StringVar(&opts.role, "role", "", opts.localizer.MustLocalize("registry.role.cmd.flag.role.description"))
 	cmd.Flags().StringVar(&opts.registryID, "instance-id", "", opts.localizer.MustLocalize("artifact.common.instance.id"))
 
+	_ = cmd.MarkFlagRequired("role")
 	_ = cmd.RegisterFlagCompletionFunc("role", func(cmd *cobra.Command, _ []string, _ string) ([]string, cobra.ShellCompDirective) {
 		return util.AllowedRoleTypeEnumValues, cobra.ShellCompDirectiveNoSpace
 	})

